manifests/claudie/terraformer.yaml

```diff
@@ -22,10 +22,10 @@ spec:
           resources:
             requests:
               cpu: 700m
-              memory: 768Mi
+              memory: 1200Mi
             limits:
               cpu: 1024m
-              memory: 1248Mi
+              memory: 1700Mi
```

Comment:
was Terraformer getting OOM-ed when this got increased? :thinking:

Reply:
Not exactly OOM-ed, more like terraform processes were stuck/error out. When I looked at the consumption of the memory in the pod, it was about 70% more than requests, thus I increased it. Now, in the latest CI, terraform ran without any errors whatsoever.